e2e/provision/README.md

```diff
@@ -364,6 +364,19 @@ kubectl get secret edge02-kubeconfig -o jsonpath='{.data.value}' | base64 -d > $
 export KUBECONFIG=$HOME/.kube/config:$HOME/.kube/regional-kubeconfig:$HOME/.kube/edge01-kubeconfig:$HOME/.kube/edge02-kubeconfig
 ```
 
+Once the edge clusters are ready it's necessary to inter-connect them. This time
+we are going to use the [containerlab tool](https://containerlab.dev/) for that
+operation.
```

Comment:
Add:

Eventually inter-cluster networking will be automated as well, but it is not yet in this release.